packages/core/src/tools.ts

```diff
@@ -249,13 +190,17 @@ export namespace Tools {
 
 	export const getProperty = (object, ...propPath) => {
```

Comment:
Why not just name it `position` to start with? :)

Reply:
because that parameter doesn't directly correspond to `position`

Reply:
Also, this is a question for @cal-smith

Reply:
I'm not sure why you would call this `position`? It gets the property for a given path (hence `getProperty`), nothing to do with positions?

We can totally change the internal variable name, I'm not not sure to what, or if that would be useful at all.